crates/bevy_tasks/src/task_pool_wasm.rs

```diff
@@ -74,14 +76,14 @@ impl TaskPool {
         let results: Mutex<Vec<Arc<Mutex<Option<T>>>>> = Mutex::new(Vec::new());
         let results: &'env Mutex<Vec<Arc<Mutex<Option<T>>>>> = unsafe { mem::transmute(&results) };
 
-        let mut scope = Scope {
+        let mut scope = PlatformScope {
             executor,
             results,
             scope: PhantomData,
             env: PhantomData,
         };
 
-        let scope_ref: &'env mut Scope<'_, 'env, T> = unsafe { mem::transmute(&mut scope) };
+        let scope_ref: &'env mut PlatformScope<'_, 'env, T> = unsafe { mem::transmute(&mut scope) };
```

Comment:
Same here.

Reply:
As you can see here this was only refactored by name, so that isn't an issue